WordPress/WordPressTest/MediaImageExporterTests.swift

```diff
@@ -174,7 +174,8 @@ class MediaImageExporterTests: XCTestCase {
         let exporter = MediaImageExporter(image: image, filename: nil)
```

Comment:
Sorry to nitpick – would you mind just updating the description for the expectation as well? Perhaps `image export by UIImage and not altering the orientation`? After that, let's :shipit: